modules/tinymce/src/core/test/ts/browser/html/SchemaTest.ts

```diff
@@ -568,6 +570,44 @@ describe('browser.tinymce.core.html.SchemaTest', () => {
     });
   });
 
+  context('TINY-10139: check elements', () => {
+    const checkElement = (name: string, predicate: (elm: SugarElement<Node>) => boolean, expectedValue: boolean) => {
+      assert.equal(predicate(SugarElement.fromTag(name)), expectedValue, `Should be ${expectedValue} for ${name}`);
+    };
+
+    const checkText = (predicate: (elm: SugarElement<Node>) => boolean) => {
+      assert.isFalse(predicate(SugarElement.fromText('text')), 'Should be false for non element');
+    };
+    it('TINY-10139: check block elements', () => {
```

Comment:
Just a blank line above this one if you are doing a bit of cleanup anyway.

```suggestion

    it('TINY-10139: check block elements', () => {
```

Reply:
done